### ISSUE TITLE
`img alt`

### ISSUE THREAD

**Issue by ghost** [malsein/f1-3-c2p1-colmar-academy#4]
https://github.com/malsein/f1-3-c2p1-colmar-academy/blob/master/capstone_colmar_assets/index.html#L26-L29

Always give your `img` tags an `alt` attribute with a value that adequately describes the image, just in case your website is not able to load it.

Doing so will help improve the accessibility of your website, which might be accessed by visually impaired users on screen readers or other devices that might deliberately neglect images-- as well as any other time that your website is just unable to render images.
